packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -1,13 +1,17 @@
 name: google_maps_flutter
 description: A Flutter plugin for integrating Google Maps in iOS and Android applications.
 homepage: https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter/google_maps_flutter
-version: 0.5.32
+version: 0.5.33
 
 dependencies:
   flutter:
     sdk: flutter
   flutter_plugin_android_lifecycle: ^1.0.0
-  google_maps_flutter_platform_interface: ^1.0.4
+  google_maps_flutter_platform_interface:
+    git:
+      url: https://github.com/tolotrasamuel/flutter_plugins.git
+      path: packages/google_maps_flutter/google_maps_flutter_platform_interface
+      ref: google_maps_gesture
```

Comment:
Please use a path-based dependency, as described [here](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changing-federated-plugins), so that it's clearer that it's testing what's in the diff.